packages/node_modules/@ciscospark/internal-plugin-metrics/src/metrics.js

```diff
@@ -26,14 +32,7 @@ const Metrics = SparkPlugin.extend({
     return this.batcher.request(Object.assign({key}, value));
   },
 
-  /**
-   * This corresponds to #sendSemiStructured() in the deprecated metrics handler
-   * @param {string} eventName
-   * @param {Object} props
-   * @param {string} preLoginId
-   * @returns {Object} HttpResponse object
-   */
-  submitClientMetrics(eventName, props, preLoginId) {
+  createPayload(eventName, props) {
```

Comment:
Please add doc block (to all new methods).

Reply:
done